### ISSUE TITLE
Add license information in csproj/Nuget

### ISSUE THREAD

**Issue by LiorBanai** [diegofrata/Generator.Equals#57]
Hi,
the nuget is missing license information so it is not shown in Visual sutdio nor in org.nuget...

you can simply add it by the following property in the csproj/:
```xml
<PropertyGroup>
  <PackageLicenseExpression>MIT</PackageLicenseExpression>
</PropertyGroup>
```
it this also recommended by microsoft:
https://learn.microsoft.com/en-us/nuget/create-packages/package-authoring-best-practices#licensing

**Comment by diegofrata** [comment]
Thanks for highlighting that. Will add this in.

**Comment by diegofrata** [comment]
Fixed in 3.1.1.
